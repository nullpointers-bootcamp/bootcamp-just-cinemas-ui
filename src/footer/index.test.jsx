import React from "react";
import renderer from "react-test-renderer";
import { BrowserRouter as Router } from "react-router-dom";
import Footer from "./index";

it("show footer", () => {
  const tree = renderer
    .create(
      <Router>
        <Footer />
      </Router>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
