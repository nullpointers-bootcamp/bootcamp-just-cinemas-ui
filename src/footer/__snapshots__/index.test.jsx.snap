// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`show footer 1`] = `
<div
  className="footer-wrapper"
>
  <a
    className="footer-links"
    href="/about-us"
    onClick={[Function]}
  >
    About Us
  </a>
  <a
    className="footer-links"
    href="/terms-and-conditions"
    onClick={[Function]}
  >
    Terms and conditions
  </a>
</div>
`;
